data_scientist_skills/data.py

```diff
@@ -0,0 +1,49 @@
+import pandas as pd
+from nltk.tokenize import word_tokenize
+from nltk.corpus import stopwords
+from nltk.stem import WordNetLemmatizer
+import string
+import unidecode
+
+def get_data(df1_dir, df2_dir): #Relative path to each data file
+    #Get data
+    df1 = pd.read_csv(df1_dir)
+    df2 = pd.read_csv(df2_dir)
+    #Drop Columns
+    df1.drop(columns='Unnamed: 0', inplace=True)
+    df2.drop(columns = ['Unnamed: 0', 'index'], inplace = True)
+    #Concat to full dataframe
+    df = pd.concat([df1, df2], ignore_index=True, axis=0)
```

Comment:
Once we have the merged dataframe, we can drop a few more columns as discussed? 

df.drop(columns = ['Revenue', 'Competitors', 'Easy Apply'], inplace= True)

'Easy Apply' refers to whether you can directly apply from Glassdoor I think, so doesn't help us very much